src/clib/pio_internal.h

```diff
@@ -79,7 +79,7 @@ extern "C" {
     int pio_num_iosystem(int *niosysid);
 
     file_desc_t *pio_get_file_from_id(int ncid);
-    int pio_get_file_from_id2(int ncid, file_desc_t **filep);
+    int pio_get_file(int ncid, file_desc_t **filep);
```

Comment:
I did replace it. Just didn't take the prototype out. But now I have.

So the end result is there is only one function to do this, pio_get_file().